tests/page-objects/messages/sms-gateway.po.js

```diff
@@ -0,0 +1,88 @@
+const utils = require('../../utils');
+const commonElements = require('../common/common.po');
+const messagePo = require('../messages/messages.po');
+const reportsPo = require('../reports/reports.po');
+const helper = require('../../helper');
+const { browser, element } = require('protractor');
+
+//page objects
+const getState = (first, second) => {
+  return element(by.css(
+    `#reports-content .scheduled-tasks > ul > li:nth-child(${first}) > ul > li:nth-child(${second}) .task-state .state`
+  ));
+};
+const  sentTask = element(by.css('#reports-content .details > ul .task-list .task-state .state'));
+const feedback = element(
+  by.css('#reports-content .details > ul .task-list .task-state .state')
+);
+
+const incomingData = element(by.css('#message-content li.incoming:first-child .data p:first-child'));
+
+// scheduled tasks
+// State for messageId2 is still forwarded-to-gateway
+
+const getTaskState = async element => {
+  await helper.waitUntilReadyNative(element);
+  return helper.getTextFromElement(element);
+};
+
+module.exports = {
+
+  showMessageList :  async () => {
+    await utils.resetBrowser();
+    await helper.clickElement(element(by.id('messages-tab')));
+
+    // LHS
+    helper.waitElementToPresent(messagePo.messageByIndex(1));
+    await browser.waitForAngular();
+    helper.waitElementToBeVisible(messagePo.messageByIndex(1));
+  },
+
+  expectMessage: async (heading, summary) => {
+    const message = messagePo.messageByIndex(1);
+    expect(await helper.getTextFromElement(message.element(by.css('.heading h4')))).toBe(heading);
+    expect(await helper.getTextFromElement(message.element(by.css('.summary p')))).toBe(summary);
```

Comment:
For here and others, please keep the `expect` calls in the tests themselves (not page objects). The reuse is admirable and most can still be retained; the change would have the test file looking something like:
```
expect(await helper.getHeading(...)).toBe(summary); 
```